app/src/main/java/com/project/lab1/feed/AddPost.kt

```diff
@@ -0,0 +1,40 @@
+package com.project.lab1.feed
+
+import android.content.Intent
+import android.os.Bundle
+import com.google.android.material.floatingactionbutton.FloatingActionButton
+import com.google.android.material.snackbar.Snackbar
+import androidx.appcompat.app.AppCompatActivity
+import com.project.lab1.R
+import com.project.lab1.feed.models.FeedItem
+
+class AddPost : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_add_post)
```

Comment:
Change init of activity view to android view binding https://developer.android.com/topic/libraries/view-binding